package.json

```diff
@@ -57,7 +60,18 @@
     "punycode": "^1.4.1",
     "rimraf": "^3.0.2",
     "semver": "^5.7.1",
-    "sinon": "^7.3.1"
+    "sinon": "^7.3.1",
+    "@babel/plugin-proposal-class-properties": "^7.1.0",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.4.4",
+    "@babel/plugin-proposal-optional-chaining": "^7.2.0",
+    "@babel/plugin-transform-modules-commonjs": "^7.0.0",
+    "@babel/preset-env": "^7.1.0",
+    "@babel/preset-flow": "^7.0.0",
+    "@babel/preset-react": "^7.0.0",
+    "babel-plugin-minify-dead-code-elimination": "^0.5.0",
+    "babel-plugin-transform-inline-environment-variables": "^0.4.3",
+    "read-pkg-up": "^4.0.0",
+    "resolve": "^1.22.1"
   },
   "engines": {
     "node": ">= 12.0.0"
```

Comment:
Should we consider moving to 18 given it's the current LTS?

Reply:
For the unpublished (dev) packages in the monorepo it doesn't matter much but Parcel does support Node >= 12 for now, so changing that would be a breaking change.

Reply:
Is this the engine supported for contributors developing parcel or for consumers of Parcel?

Reply:
This is the one for contributors.

Reply:
That's cool, thanks